rotkehlchen/api/rest.py

```diff
@@ -4416,3 +4424,29 @@ def search_for_names_everywhere(self, addresses: List[ChecksumEthAddress]) -> Re
             addresses=addresses,
         )
         return api_response(_wrap_in_ok_result(mappings))
+
+    def get_config_arguments(self) -> Response:
+        max_size_in_mb_all_logs = self.rotkehlchen.configuration['max_size_in_mb_all_logs']
+        max_num_log_files = self.rotkehlchen.configuration['max_num_log_files']
+        sqlite_instructions = self.rotkehlchen.configuration['sqlite_instructions']
+        sleep_secs = self.rotkehlchen.configuration['sleep_secs']
+        config = {
+            'max_size_in_mb_all_logs': {
+                'value': max_size_in_mb_all_logs,
+                'is_default': max_size_in_mb_all_logs == DEFAULT_MAX_LOG_SIZE_IN_MB,
+            },
+            'max_num_log_files': {
+                'value': max_num_log_files,
+                'is_default': max_num_log_files == DEFAULT_MAX_LOG_BACKUP_FILES,
+            },
+            'sqlite_instructions': {
+                'value': sqlite_instructions,
+                'is_default': sqlite_instructions == DEFAULT_SQL_VM_INSTRUCTIONS_CB,
+            },
+            'sleep_secs': {
+                'value': sleep_secs,
+                'is_default': sleep_secs == DEFAULT_SLEEP_SECS,
+            },
+        }
+        result_dict = {'result': config, 'message': ''}
+        return api_response(process_result(result_dict), status_code=HTTPStatus.OK)
```

Comment:
Is there a need to use `process_result` in this case as no serialization is happening?